test/FunctionsV2/EndToEndTraceHelperTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#nullable enable
+using System;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests;
+using Microsoft.Extensions.Logging.Abstractions;
+using Xunit;
+
+namespace WebJobs.Extensions.DurableTask.Tests.V2
+{
+    public class EndToEndTraceHelperTests
+    {
+        [Theory]
+        [InlineData(true, "DO NOT LOG ME")]
+        [InlineData(false, "DO NOT LOG ME")]
+        [InlineData(true, null)]
+        [InlineData(false, null)]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        public void StringSanitizerTest(
+            bool shouldTraceRawData,
+            string? possiblySensitiveData)
+        {
+            // set up trace helper
+            var nullLogger = new NullLogger<EndToEndTraceHelper>();
+            var traceHelper = new EndToEndTraceHelper(
+                logger: nullLogger,
+                traceReplayEvents: false, // has not effect on sanitizer
+                shouldTraceRawData: shouldTraceRawData);
+
+            // run sanitizer
+            traceHelper.SanitizeString(
+                rawPayload: possiblySensitiveData,
+                out string iLoggerString,
+                out string kustoTableString);
+
+            // expected: sanitized string should not contain the sensitive data
+            // skip this check if data is null
+            if (possiblySensitiveData != null)
+            {
+                Assert.DoesNotContain(possiblySensitiveData, kustoTableString);
+            }
+
+            if (shouldTraceRawData)
+            {
+                string expectedString = possiblySensitiveData ?? string.Empty;
+                Assert.Equal(expectedString, iLoggerString);
+            }
+            else
+            {
+                // If raw data is not being traced,
+                // kusto and the ilogger should get the same data
+                Assert.Equal(iLoggerString, kustoTableString);
+            }
+        }
+
```

Comment:
Can we remove this blank line to get rid of the warnings?

Reply:
Good catch, just did! -> https://github.com/Azure/azure-functions-durable-extension/pull/2844/commits/bac3b3376cc75c1b4d8a8f4fd10a5b4b84257b85